content/english/javascript-basics/Arrays/Methods.md

```diff
@@ -0,0 +1,51 @@
+---
+title: "Methods"
+date: 2020-07-28T11:45:38-07:00
+draft: false
+weight: 8
+---
+
+To make things easier, arrays provide a lot of methods. Let's go over some of the basic ones which would give you an idea of they look like.
+
+
+
+# Concat() 
+Joins two or more arrays and returns a new array.
+
+![#Can't find image](../img/array_methods/concat.png)
```

Comment:
I think this comment applies for the entire PR - please also have codeblocks with sample code so that it's easy to copy-paste if the students need to do so.